app/[contentType]/tags/[tag]/page.tsx

```diff
@@ -0,0 +1,36 @@
+import { Container, ContentList } from '../../../../components';
+import { getContentWithTag, IContentType } from '../../../../utils/content';
+import { contentTypesMap } from '../../../../utils/content-types';
```

Comment:
Changed the absolute imports to relative imports, because we use the Upleveled ESLint tsconfig.json and don't have paths defined

    "paths": {
      "@components": [
        "./components"
      ],
      "@utils/*": [
        "./utils/*"
      ],
    }